allennlp/data/token_indexers/wordpiece_indexer.py

```diff
@@ -245,4 +256,25 @@ def __init__(self,
                          do_lowercase=do_lowercase,
                          never_lowercase=never_lowercase,
                          start_tokens=["[CLS]"],
-                         end_tokens=["[SEP]"])
+                         end_tokens=["[SEP]"],
+                         separator_token="[SEP]")
+
+
+def _get_token_type_ids(wordpiece_ids: List[int],
+                        separator_ids: List[int]) -> List[int]:
+    length_wordpiece = len(wordpiece_ids)
+    length_end_pieces = len(separator_ids)
```

Comment:
similarly, I would call this `separator_length`, which I think is more indicative of what it actually is